pom.xml

```diff
@@ -12,7 +12,8 @@
 	</properties>
 
 	<modules>
-		<module>editorconfig-eclipse-plugin</module>
+		<module>org.eclipse.editorconfig.core</module>
```

Comment:
I would like to leave the naming conventions unchanged for the projects:

editorconfig-eclipse-core
editorconfig-eclipse-ui